core/win32/events.mc

```diff
@@ -662,4 +662,12 @@ Language=English
 Application %1!s! (%2!s!). Out of contiguous memory. %3!s!
 .
 
+MessageId =
+Severity = Error
+Facility = DRCore
+SymbolicName = MSG_AVX_512_SUPPORT_INCOMPLETE
+Language=English
+Application %1!s! (%2!s!): AVX-512 was detected at PC %3!s!
```

Comment:
IMHO this should say something about *why* it matters that AVX-512 was detected.  I think my original suggestion was "expect failure" or sthg: not sure that's great wording, but it should say something: "AVX-512 is not yet fully supported and running applications that use it is not expected to work." or some such.

Reply:
I took that sentence from you above :).
Changed.